NineChronicles.Headless/Properties/NineChroniclesNodeServiceProperties.cs

```diff
@@ -13,6 +13,10 @@ namespace NineChronicles.Headless.Properties
 {
     public class NineChroniclesNodeServiceProperties
     {
+        // A private key that is used in mining and signing transactions, which is different
+        // with the private key used in swarm to sign messages.
```

Comment:
How about commenting with XML docs? In other words, using `///` prefix.

Reply:
👍